microsoft-365/security/office-365-security/unverified-sender-feature.md

```diff
@@ -20,20 +20,20 @@ description: "To prevent phishing messages from reaching your mailbox, Outlook.c
 # Unverified Sender
 
 > [!NOTE]
-> These updates are rolling out now, and might not be available yet for all users. This feature is supported for Enterprise outlook.com users. It is not currently available for consumer outlook.com.
+> These updates are rolling out now, and might not be available yet for all users. This feature is supported for Enterprise outlook.com and Enterprise Outlook Win32 desktop users. It is not currently available for consumer office 365 users.
 
-To prevent phishing messages from reaching your mailbox, Outlook.com and Outlook on the web verify that the sender is who they say they are and mark suspicious messages as junk email.
+To prevent phishing messages from reaching your mailbox, Office 365 verifies that the sender is who they say they are and mark suspicious messages as junk email.
```

Comment:
```suggestion
To prevent phishing messages from reaching your mailbox, Office 365 verifies that senders are who they say they are and mark suspicious messages as junk email.
```